### DISCUSSION TITLE
Possible to control how multiple checkbox values are formatted when populated from a URL?

### DISCUSSION THREAD

**Discussion by jmauzyk** [verbb/formie#1216] [Q&A]
Wondering if there's currently any way to control how multiple checkbox values are formatted when populated from a URL into a form? Currently, if populated into a text field we get a format like `["abc","123"]`. Would like to use a more human-friendly format such as comma separated values if possible.

**Comment by engram-design** [comment]
Added support for this for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jmauzyk** [comment]
@engram-design still seeing checkbox values populated in form fields as an array format. Is there configuration required on our end to get this set up?

**Reply by engram-design** [reply]
@jmauzyk Might convert this to an issue and follow up there.
